windows/win_scheduled_task.py

```diff
@@ -67,6 +70,8 @@
       - The frequency of the command, not idempotent
     required: false
     choices:
+      - minutely
+      - hourly
```

Comment:
@perzizzle please make a note in the description, that the new choices `minutely`, `hourly` were added added in 2.2.